unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NotificationMessagesHUD/MainChatNotificationsComponentView.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using DCL.Interface;
+
+public class MainChatNotificationsComponentView : BaseComponentView
+{
+    [SerializeField] RectTransform chatEntriesContainer;
+    [SerializeField] GameObject chatNotification;
+
+    public static MainChatNotificationsComponentView Create()
+    {
+        return Instantiate(Resources.Load<MainChatNotificationsComponentView>("SocialBarV1/ChatNotificationHUD"));
+    }
+
+    public void AddNewChatNotification(ChatMessage message)
+    {
+        ChatNotificationMessageComponentView chatNotificationComponentView = Instantiate(chatNotification).GetComponent<ChatNotificationMessageComponentView>();
```

Comment:
I suggest you to implement a pool to manage instantiation of the notification entries. It will prevent us from being continously instantiating/removing game objects 👍